flang/test/Lower/intrinsic-procedures.f90

```diff
@@ -783,6 +783,28 @@ subroutine product_test2(a,r)
 ! CHECK: fir.call @_FortranACppProductComplex10(%[[a4]], %[[a5]], %{{.*}}, %{{.*}}, %[[a7]], %8) : (!fir.ref<complex<f80>>, !fir.box<none>, !fir.ref<i8>, i32, i32, !fir.box<none>) -> ()
 end
 
+! REPEAT
+! CHECK-LABEL: repeat_test
+! CHECK-SAME: (%[[arg0:.*]]: !fir.boxchar<1>, %[[arg1:.*]]: !fir.ref<i32>)
+subroutine repeat_test(c, n)
+  character(*) :: c
+  integer :: n
+  ! CHECK: %[[tmpBox:.*]] = fir.alloca !fir.box<!fir.heap<!fir.char<1,?>>>
+  ! CHECK-DAG: %[[c:.*]]:2 = fir.unboxchar %[[arg0]] : (!fir.boxchar<1>) -> (!fir.ref<!fir.char<1,?>>, index)
+  ! CHECK-DAG: %[[ni32:.*]] = fir.load %[[arg1]] : !fir.ref<i32>
+  ! CHECK-DAG: %[[n:.*]] = fir.convert %[[ni32]] : (i32) -> i64
+  ! CHECK-DAG: %[[cBox:.*]] = fir.embox %[[c]]#0 typeparams %[[c]]#1 : (!fir.ref<!fir.char<1,?>>, index) -> !fir.box<!fir.char<1,?>>
+  ! CHECK-DAG: %[[cBoxNone:.*]] = fir.convert %[[cBox]] : (!fir.box<!fir.char<1,?>>) -> !fir.box<none>
+  ! CHECK-DAG: %[[resBox:.*]] = fir.convert %[[tmpBox]] : (!fir.ref<!fir.box<!fir.heap<!fir.char<1,?>>>>) -> !fir.ref<!fir.box<none>>
+  ! CHECK: fir.call @{{.*}}Repeat(%[[resBox]], %[[cBoxNone]], %[[n]], {{.*}}) : (!fir.ref<!fir.box<none>>, !fir.box<none>, i64, !fir.ref<i8>, i32) -> none
```

Comment:
Please use %{{.\*}} for each ignored argument. It enforces the number of actual arguments for the call and it is easier to read. 

For example, 

Repeat(%[[resBox]], %[[cBoxNone]], %[[n]], %{{.\*}}, %{{.\*}})

Reply:
Thanks @mleair for having a look. Yes, the proposed method is better. Have made the change.